apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -178,6 +190,30 @@ const buildTokensListForCategory = (tokens, path, category, tokenType) => {
     return buildTokensList(tokenListElements);
 };
 
+/**
+ * Builds a categorised list of tokens
+ * @param {string} tokenType - the type of token i.e. color, spacing, radius
+ * @param {object} tokens
+ * @returns - object of token categories with display names i.e.  whiteBlack: { displayName: 'White and Black' }
```

Comment:
Just to double-check – does this function return an object? It looks like it returns `buildTokensList()` which outputs a string (but not sure if I'm missing something)